Tasks/ip.js

```diff
@@ -1,17 +1,3 @@
-// Split string by the first occurrence of separator
+'use strict';
 
-Parseip = i => {
-  a = []
-    if (i === '' ) return 0  ;
-  else {
-  B = i.split('.');
-  j = 0
-    for ( const b of B) {
-  a[j] = parseInt(b);
-  j++;
-  }};
-  return a;
-};
-
-const result = Parseip('127.0.0.1')
-console.log(result)
+const parseIp = ip => ip === '' ? 0 : ip.split('.').map(e => +e);
```

Comment:
- Use parseInt is preferable here
- Intermediate variables will make this code more readable
- Use round brackets even for single arguments lambda functions
- Prefer if-statement here for readability
- Improve naming, `e` is not ok

Reply:
ip.js is fully fixed